src/SprykerEco/Zed/Computop/ComputopConfig.php

```diff
@@ -416,4 +427,21 @@ public function getIdealIssuerId(): string
     {
         return $this->get(ComputopConstants::IDEAL_ISSUER_ID);
     }
+
+    /**
+     * Specification:
+     * - Get COMPUTOP payment methods with computop init payment transfer
+     *
+     * @api
+     *
+     * @return string[]
+     */
+    public function getPaymentMethodsWithInitPaymentTransfer(): array
```

Comment:
* Rename to `getPaymentMethodsWithExternalRedirect()`.

Reply:
++